Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -94,24 +94,9 @@ bash scripts/install_prerequisites.sh
 For your virtual environment, we recommend you use [pyenv](https://github.com/pyenv/pyenv). Here are some instructions for doing so, but you can use another virtual environment tool if you wish:
 
 1. **Make sure you install the Python build dependencies for your operating system. These are specified [here](https://github.com/pyenv/pyenv/wiki#suggested-build-environment). If you don't do this it might lead to problems further on.** The build dependencies for Ubuntu/Debian are
-    ```sh
-    sudo apt-get install make
-    sudo apt-get install build-essential
-    sudo apt-get install libssl-dev
-    sudo apt-get install zlib1g-dev
-    sudo apt-get install libbz2-dev
-    sudo apt-get install libreadline-dev
-    sudo apt-get install libsqlite3-dev
-    sudo apt-get install wget
-    sudo apt-get install llvm
-    sudo apt-get install libncursesw5-dev
-    sudo apt-get install xz-utils
-    sudo apt-get install tk-dev
-    sudo apt-get install libxml2-dev
-    sudo apt-get install libxmlsec1-dev
-    sudo apt-get install libffi-dev
-    sudo apt-get install liblzma-dev
-    ```
+
+
+    ```sudo apt-get install make build-essential libssl-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev wget llvm libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev```
```

Comment:
Since this is a single-line code block, you should use `` ` `` instead of ` ``` `

Reply:
@U8NWXD  I changed it from ``` to `